cpp/ql/test/query-tests/Security/CWE/CWE-367/semmle/test2.cpp

```diff
@@ -408,3 +408,8 @@ void test7_1(const char *path1, const char *path2)
 		chmod(path2, 1234); // BAD
 	}
 }
+
+int test8(const char *path, int mode)
+{
+    return (chmod(path, mode) == 0 ? 1 : 0);
```

Comment:
I'd actually like to know if this if `GOOD` or `BAD` 😅

Reply:
This looks good to me! There's a sensitive file operation here, `chmod`, which represents the "time of use" part of the query, but there's no check that guards the operation (i.e., there's no "time of check").

Reply:
Annotation added.